devel/universal-ctags/Portfile

```
@@ -21,11 +21,11 @@ long_description \
 
 homepage            https://ctags.io
 
-checksums           rmd160  e37a347213bf94639b68adacc5f66d90fe643388 \
-                    sha256  73ab06a4868f2006c2cdde2276db13f1fc010483b2711cd66ef00b51b680064d \
-                    size    1663558
+checksums           rmd160  e234b2cd0edb887df465e23bb46ecfbe17ce5c04 \
+                    sha256  db6e82f964364b829ff38ae59e49f64067f11dd9e743a8a993c9229d3ed9c0b7 \
+                    size    2024988
 
-set python_version  37
+set python_version  38
```

Should default to python 3.9 now to match fa147a0a5b020635eb792a94330a0cc5641cc7b3.
```suggestion
set python_version  39
```